doku/tests/printers/enum/of_tag/external/of_fields/named/output.with-user-comments-wrapped.commented.toml

```diff
@@ -0,0 +1,16 @@
+[value]
```

Comment:
Yes i noticed that, but i decided to implement it anyway, since it's not very different from the other formats. I don't see a point in removing it now.

Reply:
okie, sure thingo